src/Categories/Bicategory/Monad.agda

```diff
@@ -0,0 +1,26 @@
+{-# OPTIONS --without-K --safe #-}
+
+-- A Monad in a Bicategory.
+-- For the more elementary version of Monads, see 'Categories.Monad'.
+module Categories.Bicategory.Monad where
+
+open import Level
+open import Data.Product using (_,_)
+
+open import Categories.Bicategory
+open import Categories.NaturalTransformation.NaturalIsomorphism using (NaturalIsomorphism)
+
+
+record Monad {o ℓ e t} (𝒞 : Bicategory o ℓ e t) : Set (o ⊔ ℓ ⊔ e ⊔ t) where
```

Comment:
Excellent!
Maybe we should show that the definition in [Categories.Monad](https://github.com/agda/agda-categories/blob/1ba82d98faddc750fe4c0363af0d1e21292a9085/src/Categories/Monad.agda#L13) is an instance in Cat? Or even replace the latter? (Not asking you to do this, just proposing it as a future addition.)

Reply:
I think showing an equivalence is a good call, the more elementary version of monad is far easier to work with, so we probably don't want to replace it :)

Reply:
Turns out this was pretty easy! I've added a pair of proofs going both directions to the PR.

Reply:
Brilliant, thanks!